setup.py

```diff
@@ -4,134 +4,37 @@
-
-    return [convert(m) for m in REQUIREMENTS]
-
-
-def _python_lang_classifiers() -> List[str]:
-    """NOTE: Will not work after the '3.* -> 4.0'-transition."""
-    if OLDEST_PY_VERSION[0] < 3:
-        raise Exception("Python-classifier automation does not work for python <3.")
-    if PY_VERSION[0] >= 4:
-        raise Exception("Python-classifier automation does not work for python 4+.")
-
-    return [
-        f"Programming Language :: Python :: 3.{r}"
-        for r in range(OLDEST_PY_VERSION[1], PY_VERSION[1] + 1)
-    ]
-
-
-def _development_status() -> str:
-    # "Development Status :: 1 - Planning",
-    # "Development Status :: 2 - Pre-Alpha",
-    # "Development Status :: 3 - Alpha",
-    # "Development Status :: 4 - Beta",
-    # "Development Status :: 5 - Production/Stable",
-    # "Development Status :: 6 - Mature",
-    # "Development Status :: 7 - Inactive",
-    version = _get_version()
-    if version.startswith("0.0.0"):
-        return "Development Status :: 2 - Pre-Alpha"
-    elif version.startswith("0.0."):
-        return "Development Status :: 3 - Alpha"
-    elif version.startswith("0."):
-        return "Development Status :: 4 - Beta"
-    elif int(version.split(".")[0]) >= 1:
-        return "Development Status :: 5 - Production/Stable"
-    else:
-        raise Exception(f"Could not figure Development Status for version: {version}")
-
+subprocess.run(
+    "pip install git+https://github.com/WIPACrepo/wipac-dev-tools.git".split(),
+    check=True,
+)
+from wipac_dev_tools import SetupShop  # noqa: E402  # pylint: disable=C0413
 
-# Setup --------------------------------------------------------------------------------
+shop = SetupShop(
+    "rest_tools",
+    os.path.abspath(os.path.dirname(__file__)),
+    ((3, 6), (3, 8)),
+    "REST Tools in Python - Common Utilities for Client and Server",
+)
 
 setup(
-    name=NAME,
-    version=_get_version(),
     scripts=glob.glob("bin/*"),
-    packages=[
-        "rest_tools",
-        "rest_tools.client",
-        "rest_tools.server",
-        "rest_tools.utils",
-    ],
-    package_data={
-        # data files need to be listed both here (which determines what gets
-        # installed) and in MANIFEST.in (which determines what gets included
-        # in the sdist tarball)
-        #'rest_utils.server':['data/etc/*','data/www/*','data/www_templates/*'],
-    },
-    author="IceCube Collaboration",
-    author_email="developers@icecube.wisc.edu",
-    url="https://github.com/WIPACrepo/rest-tools",
-    license="https://github.com/WIPACrepo/rest-tools/blob/master/LICENSE",
-    description="REST tools in python - common code for client and server ",
-    long_description=open(os.path.join(HERE, "README.md")).read(),  # include new-lines
-    long_description_content_type="text/markdown",
-    classifiers=sorted(
-        _python_lang_classifiers()
-        + [_development_status()]
-        + [
+    url="https://github.com/WIPACrepo/wipac-telemetry-prototype",
```

Comment:
Wait a minute... this doesn't seem like the right URI for `rest_tools` :thinking:

Reply:
d'oh!